src/Pages/kiwanlee/Login/Login.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import './Login.scss';
+//import { Link } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputId: '',
+      inputPw: '',
+      background: '#bfdffd',
+      disabled: true,
+      idChecked: false,
+      pwChecked: false,
+    };
+  }
+
+  handleInputId = e => {
+    this.setState({ inputId: e.target.value });
+    e.target.value.includes('@')
+      ? this.setState({ idChecked: true }, () => this.btnChangeColor())
+      : this.setState({ idChecked: false }, () => this.btnChangeColor());
+  };
+
+  handleInputPw = e => {
+    this.setState({ inputPw: e.target.value });
+    e.target.value.length > 4
+      ? this.setState({ inputPw: e.target.value, pwChecked: true }, () =>
+          this.btnChangeColor()
+        )
+      : this.setState({ pwChecked: false }, () => this.btnChangeColor());
+  };
+
+  btnChangeColor = () => {
+    this.state.idChecked && this.state.pwChecked
+      ? this.setState({ background: '#1c7ed6', disabled: false })
+      : this.setState({ background: '#bfdffd', disabled: true });
+  };
+  goToMain = () => {
+    fetch('http://10.58.6.180:8000/authenticate/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.inputId,
+        password: this.state.inputPw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.token) {
+          this.props.history.push('/mainW');
+        }
+      });
+  };
+
+  render() {
+    console.log(this.state);
+    return (
+      <body>
```

Comment:
```suggestion

  render() {
    let validation = this.state.inputId.includes('@') && this.state.inputPw.length > 4
    return (
      <body>
```
이런식으로 render 함수 밑에  validation 이라는 변수를 생성하고, 사용해보세요!!
기존 방식 처럼 state로 색상변경이나 버튼 활성화를 하나 하나 해줄 필요가 없습니다!